src/ResourceManager/IotHub/Commands.IotHub/Models/PSCertificateListDescription.cs

```diff
@@ -0,0 +1,30 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Management.IotHub.Models
```

Comment:
This model seems unnecessary.  Why don't you just receive a list of CertificateDescriptions and convert them to PSCertificateDescriptions?

Reply:
Removed